apps/src/code-studio/initApp/renderPrivacyProfanityViolating.js

```diff
@@ -0,0 +1,24 @@
+/* global dashboard */
+import React from 'react';
+import ReactDOM from 'react-dom';
+var AbuseExclamation = require('../components/abuse_exclamation');
+var showProjectAdmin = require('../showProjectAdmin');
```

Comment:
Ah, this all was copied from the abuse version of this component and not updated for ES6, was on the fence about introducing `tos` as a parameter to that one or whether these two components may diverge more in the future, in which case keeping them separate might avoid parameter explosion. This version probably shouldn't have the `showProjectAdmin` either.

Thinking through it I'll keep them separate and update this file to ES6 as it's new.